lib/render/vp2RenderDelegate/proxyRenderDelegate.h

```diff
@@ -142,7 +142,7 @@ class ProxyRenderDelegate : public MHWRender::MPxSubSceneOverride
     void _FilterSelection();
     void _UpdateSelectionStates();
 
-    const MayaUsdProxyShapeBase*  _proxyShape{ nullptr }; //!< DG proxy shape node
```

Comment:
Loss of constness seems like a shame.

Reply:
Indeed...but we will need a better solution to these new callbacks, so we will get back constness here.